conformance/utils/suite/suite.go

```diff
@@ -261,6 +263,22 @@ func ParseNamespaceLabels(f string) map[string]string {
 	return res
 }
 
+// ParseNamespaceAnnotations parses flag arguments and converts the string to
+// map[string]string containing annotation key/value pairs.
+func ParseNamespaceAnnotations(f string) map[string]string {
```

Comment:
Why not just rename `ParseNamespaceLabels` something like `ParseKeyValuePairs` and use it for both flags, instead of having two functions with the same impl?

Reply:
Being honest, I think I was expecting to have to edit it after the clone, and then never needed to. Great suggestion, I'll change that.